app/src/main/java/com/example/android/brujulalogica/SignUpScreen.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.android.brujulalogica
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import com.google.firebase.auth.FirebaseAuth
+import kotlinx.android.synthetic.main.sign_up_screen.*
+
+class SignUpScreen : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.sign_up_screen)
+
+        signIn ()
+    }
+
+    private fun signIn() {
```

Comment:
devuelta el nombre aca es engañoso, no estas haciendo un sign in estas seteando un sign in